vision/cloud-client/face_detection/faces.py

```diff
@@ -18,14 +18,14 @@
 
 import argparse
 
-# [START import_client_library]
+# [START vision_face_detection_tutorial_imports]
 from google.cloud import vision
-# [END import_client_library]
 from google.cloud.vision import types
 from PIL import Image, ImageDraw
+# [END vision_face_detection_tutorial_imports]
 
 
-# [START def_detect_face]
+# [START vision_face_detection_tutorial_send_request]
```

Comment:
`send_request` is a bit vague/generic.

Reply:
It's within the context of being the face detection tutorial, so I think it's an ok suffix. This was done to standardize across languages. `def_detect_face` is too vague because there is also a face detection sample for the how to guides.